pkg/k8s/factory_functions.go

```diff
@@ -8,7 +8,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	"github.com/cilium/cilium/pkg/comparator"
-	"github.com/cilium/cilium/pkg/datapath"
+	types2 "github.com/cilium/cilium/pkg/datapath/types"
```

Comment:
Maybe something like `dptypes` instead? Not sure why this import name was chosen. We should also try to use the same import name across all consuming packages.

Reply:
curious on how we can enforce this. I think right now, it's just manual effort only. However, it's not uncommon in cilium code base.

```
nodeTypes "github.com/cilium/cilium/pkg/node/types"
cnitypes "github.com/containernetworking/cni/pkg/types"
eniTypes "github.com/cilium/cilium/pkg/aws/eni/types"
ipcachetypes "github.com/cilium/cilium/pkg/ipcache/types"
```

Reply:
@ti-mo Ping?